multiclusterhub/charts/grc/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 Red Hat, Inc.
+
+apiVersion: v1
+appVersion: "1.0.0"
+description:  helm chart for multicloud grc
+name: grc
+version: 2.1.0
+category: "Development"
+verified: "RHACM"
+keywords:
+  - acm
+  - grc
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822